pypi-vuln/index-2021-07-27-role-deletion.rst

```diff
@@ -0,0 +1,71 @@
+======================================
+Vulnerability in Role Deletion on PyPI
+======================================
+
+An exploitable vulnerability in the mechanisms for deleting roles on `PyPI
+<https://pypi.org>`_ was discovered by a security researcher, which would allow
+an attacker to remove roles for projects not under their control.
+
+* Disclosure date: **2020-07-25** (Reported via security policy on `pypi.org <https://pypi.org/security/>`_)
```

Comment:
I think this date should be `2021-07-26` (I reported it on `2021/07/26 22:57` UTC)